common/api/core-backend.api.md

```diff
@@ -2905,6 +2905,9 @@ export abstract class IModelDb extends IModel {
     // @internal (undocumented)
     protected _codeService?: CodeService;
     get codeSpecs(): CodeSpecs;
+    // @beta
```

Comment:
is it beta or public? what about the setter?

Reply:
I changed it to public on his original PR. @DanRod1999 does PR validation validate extract-api?